artemis-hawtio/artemis-plugin/src/main/webapp/plugin/html/consumers.html

```diff
@@ -44,9 +44,30 @@
                     </button>
                 </form>
             </div>
+
+            <div class="pull-right">
+              <form class="form-inline">
+                <button class="btn-danger" ng-disabled="gridOptions.selectedItems.length == 0"
+                        ng-click="deleteDialog = true"
+                        title="Close the selected Consumer">
+                  <i class="icon-remove"></i> Close
+                </button>
+                &nbsp;&nbsp;&nbsp;
+              </form>
+            </div>
         </div>
     </div>
 
+    <div hawtio-confirm-dialog="deleteDialog"
+          ok-button-text="Close"
+          on-ok="closeConsumer()">
+       <div class="dialog-body">
+         <p>You are about to close the selected session: {{gridOptions.selectedItems[0].id}}
```

Comment:
Actually, the button does seem to be disabled.  It just isn't visibly disabled, e.g. greyed out etc... I think this is OK.  I have some minor layout fixes and the comment from @michaelandrepearce .  I will squash and merge.